src/main/java/org/ow2/proactive/connector/iaas/cloud/provider/jclouds/JCloudsProvider.java

```diff
@@ -444,4 +442,21 @@ private Set<NodeCandidate> getPaidNodeCandidate(Infrastructure infra, String reg
         }).collect(Collectors.toSet());
         return result;
     }
+
+    /**
+     * @return The Operating system family based on the infrastructure type, created to patch the limitation caused
+     * by Jclouds for OpenStack (not taking the distro_family into account).
+     */
+    private String getOperatingSystemFamily(org.jclouds.compute.domain.Image image, String infrastructureType) {
+        if (!infrastructureType.equals("openstack-nova")) {
+            return Optional.ofNullable(image.getOperatingSystem().getFamily())
+                           .orElse(org.jclouds.compute.domain.OsFamily.UNRECOGNIZED)
```

Comment:
same as above

Reply:
noted! ;)

Reply:
again org.jclouds.compute.domain.OsFamily and org.jclouds.scriptbuilder.domain.OsFamily so I can't